### ISSUE TITLE
The mIoU is low

### ISSUE THREAD

**Issue by jackieyung** [dong-x16/PortraitNet#10]
Thanks for your sharing @dong-x16 . 

I have an issue. When I train my model without the prior channel, the mIoU of the model can reach 94%, but when I add prior channel, the mIoU of the model is much lower than without the prior channel, which even cannot reach 90%. 

Could you give me a piece of advice about how to set the parameters of every loss? 

Thanks a lot.

**Comment by ccly1996** [comment]
hello，excuse me, could you display your requirements details?thank you

**Comment by jackieyung** [comment]
> hello，excuse me, could you display your requirements details?thank you

Hi, I just need to improve the mIoU of the model whose input has the prior channel. But I find when I add the fourth prior channel, the mIoU of the model drops a lot.

**Comment by ccly1996** [comment]
emmm, I am deploying this code, but wrong happened when i install requirements, can you help me?The author may write something wrong in requirements, so i need requirements list.😂

**Comment by jackieyung** [comment]
> emmm, I am deploying this code, but wrong happened when i install requirements, can you help me?The author may write something wrong in requirements, so i need requirements list.😂

python 2.7
PyTorch 0.3.0.post4
Jupyter Notebook
pip install easydict matplotlib tqdm opencv-python scipy pyyaml numpy

**Comment by ccly1996** [comment]
thanks,i install these requirements ,but something wrong.

**Comment by jackieyung** [comment]
> thanks,i install these requirements ,but something wrong.

Could you paste your detailed wrong message?

**Comment by ccly1996** [comment]
the wrong is from torchvision, when i import torchvision.models.i think this wrong from the version of torchvision, but the author did not note that.

**Comment by decajcd** [comment]
> Thanks for your sharing @dong-x16 .
> 
> I have an issue. When I train my model without the prior channel, the mIoU of the model can reach 94%, but when I add prior channel, the mIoU of the model is much lower than without the prior channel, which even cannot reach 90%.
> 
> Could you give me a piece of advice about how to set the parameters of every loss?
> 
> Thanks a lot.

Have you resolved it?

**Comment by shivSD** [comment]
@jackieyung @dong-x16 
When you are training with Prior are you using Images (EG1800 or Supervisely) or video, Correct me if i'm wrong while training with Prior channel network will be receiving previous mask information, So during training, there should be some correction between the current frame and previous frame right, we need to train the prior with video.

**Comment by dpcq** [comment]
> Thanks for your sharing @dong-x16 .
> 
> I have an issue. When I train my model without the prior channel, the mIoU of the model can reach 94%, but when I add prior channel, the mIoU of the model is much lower than without the prior channel, which even cannot reach 90%.
> 
> Could you give me a piece of advice about how to set the parameters of every loss?
> 
> Thanks a lot.

hello  when you train you model ,how many epoch do you choose? i have to train 2000 epoch to get miou 95%, do you have the question
